Rakefile

```diff
@@ -8,11 +8,7 @@ RSpec::Core::RakeTask.new(:specs)
 
 task default: :specs
 
-task :spec do
-  Rake::Task["specs"].invoke
-  Rake::Task["rubocop"].invoke
-  Rake::Task["spec_docs"].invoke
-end
+task spec: [:specs, :rubocop, :spec_docs]
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/manicmaniac/danger-periphery/blob/a709dc0e0fef8439058149b9084698dbe4bdf8ca/Rakefile#L11"></span>Use <code>%i</code> or <code>%I</code> for an array of symbols.</td>
    </tr>
  </tbody>
</table>

Reply:
Fixed in https://github.com/manicmaniac/danger-periphery/pull/3/commits/2e7f34e6e1d85676b736d9a3024f73a434c5ec0b